qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/files/FilesFragmentUITest.java

```diff
@@ -100,38 +106,15 @@ public String getName() {
 
     }
 
-    public FilesFragmentUITest() throws Exception {
-        //setup data before MainActivity launched. This avoid the call to create identity
-        if (mFillAccount) {
+    public FilesFragmentUITest(){
+        try {
             setupData();
+        } catch (Exception e) {
+            Log.e(TAG, "Cannot setupData", e);
```

Comment:
Pleae don't put setup code in the constructor. If you need to run setup code before the activity is launched, just set the IntentsTestRule to not launch automatically (last argument to 'false'), use proper @beforeClass methods and launch the activity in the setUp method with testRule.launchActvity(null);